environments/neutron-opendaylight-l2gw-sriov.yaml

```diff
@@ -0,0 +1,35 @@
+# A Heat environment that can be used to deploy OpenDaylight with L2GW and SR-IOV enabled
+resource_registry:
+  OS::TripleO::Services::NeutronOvsAgent: OS::Heat::None
+  OS::TripleO::Services::ComputeNeutronOvsAgent: OS::Heat::None
+  OS::TripleO::Services::ComputeNeutronCorePlugin: OS::Heat::None
+  OS::TripleO::Services::OpenDaylightApi: ../puppet/services/opendaylight-api.yaml
+  OS::TripleO::Services::OpenDaylightOvs: ../puppet/services/opendaylight-ovs.yaml
+  OS::TripleO::Services::NeutronL3Agent: OS::Heat::None
+  OS::TripleO::Services::NeutronSriovAgent: ../puppet/services/neutron-sriov-agent.yaml
+
+parameter_defaults:
+  NeutronEnableForceMetadata: true
+  NeutronMechanismDrivers: ['sriovnicswitch','openvswitch','opendaylight_v2']
+  NeutronServicePlugins: "odl-router_v2,networking-l2gw"
+  NeutronNetworkType: 'vxlan'
+  NeutronTenantNetworkType: vlan
+  NeutronNetworkVLANRanges: physnet1:3001:3009
+  ExtraConfig:
+neutron::server::service_providers: [ 'L2GW:OpenDaylight:networking_odl.l2gateway.driver.OpenDaylightL2gwDriver:default' ]
+
+  #enable DHCP on ODL
+  opendaylight::enable_dhcp: true
+
+  # SR-IOV
+  # Add PciPassthroughFilter to the scheduler default filters
+  NovaSchedulerDefaultFilters: ['RetryFilter','AvailabilityZoneFilter','RamFilter','ComputeFilter','ComputeCapabilitiesFilter','ImagePropertiesFilter','ServerGroupAntiAffinityFilter','ServerGroupAffinityFilter','PciPassthroughFilter']
+  NovaSchedulerAvailableFilters: ["nova.scheduler.filters.all_filters","nova.scheduler.filters.pci_passthrough_filter.PciPassthroughFilter"]
+  # Provide the vendorid:productid of the PF,VFs
+  NeutronSupportedPCIVendorDevs: ['8086:10ed','8086:10fb']
+  NeutronPhysicalDevMappings: "physnet1:em2"
+  # Number of VFs that needs to be configured for a physical interface
+  NeutronSriovNumVFs: "em2:8"
```

Comment:
shouldn't this be a per deployment setting? since nic names can change based on deployment